src/frequenz/dispatch/__init__.py

```diff
@@ -0,0 +1,72 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""A highlevel interface for the dispatch API."""
+
+import grpc
+from frequenz.channels import Broadcast, Receiver
+from frequenz.client.dispatch.types import Dispatch
+
+from frequenz.dispatch.actor import DispatchActor
+
+__all__ = ["Dispatcher"]
+
+
+class Dispatcher:
+    """A highlevel interface for the dispatch API.
+
+    This class provides a highlevel interface to the dispatch API. It
+    allows to receive new dispatches and ready dispatches.
+
+    Here is an example of how to use the `Dispatcher` class:
+
+    ```python
+    from frequenz.dispatch import Dispatcher
+
+    async def run():
+        dispatcher = Dispatcher(API_CONNECTION_INFO)
+        dispatcher.start()  # this will start the actor
+        dispatch_arrived = dispatcher.new_dispatches().new_receiver()
+        dispatch_ready = dispatcher.ready_dispatches().new_receiver()
+    ```
```

Comment:
```suggestion
    Example:
        ```python
        from frequenz.dispatch import Dispatcher

        async def run():
            dispatcher = Dispatcher(API_CONNECTION_INFO)
            dispatcher.start()  # this will start the actor
            dispatch_arrived = dispatcher.new_dispatches().new_receiver()
            dispatch_ready = dispatcher.ready_dispatches().new_receiver()
        ```
```